kodi_addon_checker/check_files.py

```diff
@@ -135,6 +135,9 @@ def check_file_permission(report: Report, file_index: list):
         :file_index: list having names and path of all the files present in addon
     """
 
+    if os.name == "nt":
```

Comment:
Shouldn't the check be done outside of the method? Or decorate it with a windows only helper wrapper...
Seems silly to enter the method just to return early. In python it might be seen as a bad practice or non-pythonic.
Either way I don't use windows and I am not the best person to review this

Reply:
I had it outside, but changed it. If we included it in the calling method, it should be 'aware' of the functionality inside the `check_file_permission` method and from an Object Orientation perspective I would say that it should not. That is why I kept it inside: `check_file_permission` actually knows what is being validated and can decide it is not needed for Windows.

But hey, one way or the other works for me.

Reply:
why don't you anotate the method with something like a `@posix_only` decorator that would check `os.name` or `sys.platform` and in case of windows return early otherwise enter the method?

Reply:
Interesting, I will have a look at it tonight.

Reply:
I added a decorator to it (after some discussion with you on Slack)